src/libs/actions/IOU.js

```diff
@@ -485,36 +500,28 @@ function getMoneyRequestInformation(report, participant, comment, amount, curren
  * Requests money based on a distance (eg. mileage from a map)
  *
  * @param {Object} report
- * @param {String} payeeEmail
- * @param {Number} payeeAccountID
  * @param {Object} participant
  * @param {String} comment
- * @param {Object[]} waypoints
- * @param {String} waypoints[].address required and must be non empty
- * @param {String} [waypoints[].lat] optional
- * @param {String} [waypoints[].lng] optional
  * @param {String} created
  * @param {String} [transactionID]
  */
-function createDistanceRequest(report, payeeEmail, payeeAccountID, participant, comment, waypoints, created, transactionID) {
-    const {payerEmail, iouReport, chatReport, transaction, iouAction, createdChatReportActionID, createdIOUReportActionID, reportPreviewAction, onyxData} = getMoneyRequestInformation(
+function createDistanceRequest(report, participant, comment, created, transactionID) {
+    const {iouReport, chatReport, transaction, iouAction, createdChatReportActionID, createdIOUReportActionID, reportPreviewAction, onyxData} = getMoneyRequestInformation(
         report,
         participant,
         comment,
         0,
         'USD',
         created,
         '',
-        payeeAccountID,
-        payeeEmail,
+        null,
+        null,
```

Comment:
This changed caused #26722. As a fix, we set current user id and email as they'll be payee.

<img width="494" alt="Screenshot 2023-10-09 at 11 25 55 AM" src="https://github.com/Expensify/App/assets/108292595/806dec4b-f7d8-4871-850d-4db7f72ca83b">